testing/tests/DevExpress.ui.widgets/treeViewParts/focusing.js

```diff
@@ -75,3 +75,61 @@ QUnit.test("focus on the item should move scroll position to this item (T226868)
     $item.trigger("dxpointerdown");
 
 });
+
+QUnit.test("Checkbox should be clicked when treeview in not in focus", function(assert) {
+    var data = [
+        { id: 1, parentId: 0, text: "Cats" },
+        { id: 2, parentId: 0, text: "Dogs" },
+        { id: 3, parentId: 0, text: "Tigers" },
+        { id: 4, parentId: 0, text: "Birds" },
+        { id: 5, parentId: 0, text: "Hippos" },
+        { id: 6, parentId: 0, text: "Elefants" },
+        { id: 7, parentId: 0, text: "Cows" }
+    ];
+
+    var $treeView = initTree({
+        dataSource: data,
+        dataStructure: "plain",
+        showCheckBoxesMode: "normal",
+        height: 100
+    });
+
+    var treeView = $treeView.dxTreeView("instance");
+    var pointerDownStub = sinon.stub(treeView, "_itemPointerDownHandler");
+
+    var $node = $treeView.find("." + internals.NODE_CLASS).eq(5),
+        $checkBox = $node.find(".dx-checkbox");
```

Comment:
This will trigger event on each checkbox of the node and it's children nodes. Use more concrete selector